src/hostname/hostnamed.c

```diff
@@ -430,6 +430,32 @@ static int context_write_data_machine_info(Context *c) {
         return write_env_file_label("/etc/machine-info", l);
 }
 
+// https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/sound/soc/soc-core.c?h=v5.10#n1476
```

Comment:
we avoid C++ comments (i.e. //), and stick to C comments (i.e. /* */), see coding style:

https://systemd.io/CODING_STYLE/#formatting

// are left for "FIXME" style comments, for documenting comments please use only /* */